rdr_service/model/participant_summary.py

```diff
@@ -333,7 +334,8 @@ def organizationId(cls):
     def siteId(cls):
         return Column("site_id", Integer, ForeignKey("site.site_id"))
 
-    consentCohort = Column("consent_cohort", Enum(ParticipantCohort), nullable=True, default=0)
+    consentCohort = Column("consent_cohort", Enum(ParticipantCohort), nullable=True)
```

Comment:
The `default = 0` is missing from the `consentCohort` column, was this intentional?

Reply:
Thanks for catching.  Was not intentional.   Will fix with the alembic change.

Reply:
Latest changes committed.  Per our offline conversation, used the ParticipantCohort.UNSET instead of 0 for the default.